python/requirements.txt

```diff
@@ -13,4 +13,10 @@ nest_asyncio
 prompt_toolkit
 mock
 pytest-asyncio
-lastmile_utils==0.0.3
\ No newline at end of file
+lastmile_utils==0.0.3
+transformers
+
+#Pytorch
+torch
+torchvision
+torchaudio
```

Comment:
just a note that the pyproject.toml isn't dynamically linked to the requirements.txt (this can be changed). If you introduce dependencies make sure to update the pyproject accordingly

Reply:
Fixed in https://github.com/lastmile-ai/aiconfig/pull/358